app.config.ts

```diff
@@ -66,5 +81,6 @@ export default ({ config }: any): any => {
         },
       ],
     },
+    plugins: ["sentry-expo"],
```

Comment:
need to specify sentry as a plugin https://docs.expo.dev/guides/using-sentry/#42-add-the-config-plugin